src/commands/force/user/password/generate.ts

```diff
@@ -26,6 +27,14 @@ export class UserPasswordGenerateCommand extends SfdxCommand {
       char: 'o',
       description: messages.getMessage('flags.onBehalfOf'),
     }),
+    length: flags.number({
+      char: 'l',
+      description: messages.getMessage('flags.length'),
+    }),
+    complexity: flags.number({
+      char: 'c',
```

Comment:
definitely have min/max here (and a default!).  Also, maybe `integer` an not `number` (because 1.5 would be allowed but bad).

oclif/sfdx flags help a lot with restricting user input and automatically giving nice help and messages when they do it wrong.